### ISSUE TITLE
Use with Service Worker Asset Cache

### ISSUE THREAD

**Issue by broerse** [martndemus/ember-font-awesome#201]
I added to ember-cli-build.js
```
// Add fonts to Service Worker asset cache
    'asset-cache': {
      include: [
       'assets/**/*',
       'fonts/**.*'
      ],
    },
```
but fonts can't be found in offline mode because '?v=4.7.0' is added to the .css (fontawesome-webfont.woff2?v=4.7.0) 
Can this be disabled or should I write something different then `'fonts/**.*'` ?

**Comment by jrjohnson** [comment]
@broerse this is an issue with fontawesome itself and I don't think we can disable it here. Since the asset is fingerprinted by the version number in the file name I use the cache-first strategy. I have my service worker definition setup as:

```
'esw-cache-first': {
  patterns: [
    'fonts/fontawesome(.+)',
  ]
},
```

**Comment by broerse** [comment]
@jrjohnson Thanks! Wil use that for now but it feels wrong to use an extra addon just for fingerprinting. Perhaps @martndemus can fix this in ember-font-awesome v4.0.0 because everything seems to be compiled in this version.

**Comment by jrjohnson** [comment]
You should be able to use this same pattern with `asset-cache`, I just use cache first so there isn't an extra network request for a file that never changes. Service worker addons are super lightweight so I wouldn't worry about it too much.  Either way should work though.

This problem *is* addressed in Font Awesome 5 and we're working on making that available.
